frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarTaskQueue.tsx

```diff
@@ -18,7 +18,7 @@ const EmptyTaskQueue = (): React.ReactElement => (
 
 function renderTaskList(tasks: readonly Task[]): ReactNode {
   return [...tasks].sort(sortTask).map((task) => {
-    return <GroupTask key={task.id} original={task} memberName="" />;
+    return <GroupTask key={task.id} original={task} memberName="" groupID="" memberEmail="" />;
```

Comment:
this doesn't look right...

Reply:
this is this 
![image](https://user-images.githubusercontent.com/33106747/100286291-8cb4be80-2f40-11eb-8a7a-a2b4be9caf83.png)
right? What is the issue?

Reply:
I don't think passing these dummy data is a good idea. What's your reason of doing this?

Reply:
I copied the format from the memberName prop

Reply:
hmm the original code also doesn't have good practice... If you can explain to me why this is safe I can let this go through, but there needs to be a follow up task to clean this up.

Reply:
Well, this belongs to the personal group task queue and it doesn't make sense to notify yourself about your own tasks, hence there is no bell icon on these tasks-- i.e. it doesn't necessarily need either of these fields. Although, I do also think that it is weird to use a GroupTask here since such is the case.